scripts/core/services/preferencesService.ts

```diff
@@ -89,7 +89,7 @@ export default angular.module('superdesk.core.preferences', ['superdesk.core.not
                         if ('Notification' in window && Notification.permission !== 'denied') {
                             Notification.requestPermission((permission) => {
                                 if (permission === 'granted') {
-                                    new Notification(msg);
+                                    return new Notification(msg);
```

Comment:
Why have you added the return?

Reply:
It was complaining about the value not being used, so I "used" it. It's not complaining now for some reason so I'll change it back.

Reply:
Yeah, it may look like it to a static analyzer, because you are creating a class instance and not using it. It's not obvious it's doing side effects in the constructor.

Reply:
I was wrong, it was `tslint` complaining about it, I put it back

Reply:
I think adding a ignore for tslint is better. It's harder to change code when there are returns which's values aren't being used but you still need to check all the places calling it if you are refactoring something.